prime-router/metadata/hl7_mapping/OML_O21/OML_O21-base.yml

```diff
@@ -66,3 +66,10 @@ elements:
       hl7OBXField: '%{hl7SegmentGroup}/OBX'
     resourceIndex: resultIndex
 
+  - name: order-specimen
+    resource: Bundle.entry.resource.ofType(Specimen)
+    condition: '%resource.count() = 1'
```

Comment:
Nit: this could just be `%resource.exists()`.  Is there a situation where the FHIR would contain more than one `Specimen` and you only want the first?

Reply:
We are only expecting one specimen resource.

Reply:
At this time we are only expecting to receive 1 `SPECIMEN` in a FHIR bundle.